client/utils/networking.ts

```diff
@@ -0,0 +1,8 @@
+import axios from 'axios';
+
+const API_BASE_URL = 'http://49.50.173.188/api';
+
+export const getAllArtworks = (userId: number) => {
+    const result = axios.get(`${API_BASE_URL}/users/${userId}/artworks`);
+    return result;
```

Comment:
불 필요한 변수할당 제거 하는게 어떨까?
```suggestion
    return axios.get(`${API_BASE_URL}/users/${userId}/artworks`);
```